packages/client/src/lib/interface.ts

```diff
@@ -75,6 +75,18 @@ export type PutOptions = {
    * Maximum times to retry a failed upload.
    */
   maxRetries?: number
+  /**
+   * Should input files be wrapped with a directory. 
+   * Enabling it preserves the input filenames in DAG as they become
+   * directory entries in the generated wrapping dir. 
+   * 
+   * The trade off is your root CID will be that of the wrapping dir, 
+   * rather than the input file itself. 
+   * 
+   * so for a single file. e.g. `cat.png` its ipfs path would be 
+   * `<wrapping dir cid>/cat.png` rather than just `<cid for cat.png>`
```

Comment:
```suggestion
   * `<wrapping dir cid>/cat.png` rather than just `<cid for cat.png>`
   *
   * Wrapping with a directory is required when passing multiple files
   * that do not share the same root.
```